lisp/main.lisp

```diff
@@ -1,28 +1,28 @@
 (in-package #:mahogany)
 
+(defvar *user-config-files*
+  (list (merge-pathnames #p"mahogany/init.lisp" (uiop:xdg-config-home))
```

Comment:
This has the same problem as mentioned in #155; if the user changes the config home location as a part of troubleshooting, then it won't take effect until they reassign this variable or restart, which might not be what they want.

Reply:
Thanks for your thoughts! I replied there too but it may worth repeating here again. I think we need to decide our target audience / users. I think most of our users will be (newbie or experienced) lispers. They should eventually know what's good to have such a dynamic environment.

One drawback is that since we're using the whole language (instead of a DSL) to configure our program, we shouldn't expect the config to be idempotent. So we certainly don't want the config to be evaluated at anytime. 

Once the user has hopped into the running WM, their best bet is to hack in the repl and evaluate whatever they want. - **They shouldn't expect things to change right after they edit their configs**.